### PR TITLE
Feature/add reducers and actions

### PR DESCRIPTION
**Pull request by AckonSamuel**
## Description:
The Redux Store was configured in the milestone, and actions and reducers for adding and removing books were implemented.

### General requirements:

- [x]  Make sure that there are no linter errors.
- [x]  Make sure that you used correct flow Gitflow
- [x]  Make sure that you documented your work in a professional way

### Milestone requirements

- [x]  Add React Redux (` npm install react-redux`)
- [x]  Structure your application's files using a "feature folder" approach and use the ducks pattern for your Redux files.
- [x]  Write your book's actions and reducer (in `/src/redux/books/books.js`) using the ducks pattern.
- [x] Write your categories actions and reducer (in `/src/redux/categories/categories.js`) using the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- [x] Configure the Redux Store (/src/redux/configureStore.js)

<hr>
 Happy Reviewing 🐣

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (mmsesay) [inline src/redux/books/books.js]: - [x] Please use the ducks' pattern in defining the action types.

See example:
```js
const ADD_BOOK = 'bookstore/books/ADD_BOOK';
const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (mmsesay) [inline src/redux/categories/categories.js]: - [x] Please use the ducks' pattern here as well.

See example:
```js
const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @AckonSamuel,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AckonSamuel** [comment]
Thank you for the review. I have implemented the changes as requested 😄

**Review by keroloussamy — APPROVED** [review]
Hi @AckonSamuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**